nessie/src/main/java/org/apache/iceberg/nessie/NessieTableOperations.java

```diff
@@ -55,6 +55,8 @@ public class NessieTableOperations extends BaseMetastoreTableOperations {
    */
   public static final String NESSIE_COMMIT_ID_PROPERTY = "nessie.commit.id";
 
+  public static final String NESSIE_GC_WARNING_PROPERTY = "nessie.gc.user.warned";
```

Comment:
Should this maybe be something like `nessie.gc.no-warning`?

Reply:
I like this name more as well :)